src/commands/createWebApp/setDefaultRgPlanNameLocation.ts

```diff
@@ -9,55 +9,71 @@ import { AppServicePlan } from "azure-arm-website/lib/models";
 import { MessageItem } from "vscode";
 import { IAppServiceWizardContext, SiteNameStep, WebsiteOS } from "vscode-azureappservice";
 import { createAzureClient, DialogResponses, IActionContext } from "vscode-azureextensionui";
+import { LocationListStep } from 'vscode-azureextensionui';
 import { ext } from "../../extensionVariables";
+import { getResourceGroupFromId } from '../../utils/azureUtils';
 import { nonNullProp } from "../../utils/nonNull";
 import { getWorkspaceSetting, updateGlobalSetting } from "../../vsCodeConfig/settings";
+import { AzConfig, AzConfigProperty, readAzConfig } from "./readAzConfig";
 
 const maxNumberOfSites: number = 3;
 
-export async function setDefaultRgAndPlanName(wizardContext: IAppServiceWizardContext, siteNameStep: SiteNameStep): Promise<void> {
-    // this should always be set when in the basic creation scenario
+export async function setDefaultRgPlanNameLocation(wizardContext: IAppServiceWizardContext, siteNameStep: SiteNameStep): Promise<void> {
```

Comment:
Not a big fan of this function/file name because we'll have to change it every time we change which defaults are included in this. Also it's confusing how we have this _and_ `setAppWizardContextDefault` and the names don't line up at all.

A few ideas:
- Option 1
    - setAppWizardContextDefault -> setPrePromptDefaults
    - setDefaultRgPlanNameLocation -> setPostPromptDefaults
- Option 2
    - setAppWizardContextDefault -> setCreateDefaults
    - setDefaultRgPlanNameLocation -> setBasicCreateDefaults
- Option 3
    - setAppWizardContextDefault -> setWebAppRecommendations
    - setDefaultRgPlanNameLocation -> setWebAppDefaults